Entity.java

```diff
@@ -0,0 +1,74 @@
+public class Entity {
```

Comment:
Let's add a constructor to the class, you can add a simple one for now with only the name as a requirement, while the rest of the parameters can take dummy values